app/javascript/components/admin/instances/card.js

```diff
@@ -0,0 +1,105 @@
+import Deploy from './actions/deploy'
+import Restart from './actions/restart'
+import Stop from './actions/stop'
+import Delete from './actions/delete'
+import axios from 'axios'
+
+export default {
+  name: 'InstanceCard',
+
+  props: {
+    instance: Object
+  },
+
+  methods: {
+    updateStatus (data) {
+      this.status = data.status
+      this.processing = data.processing
+    }
+  },
+
+  data () {
+    return {
+      status: this.instance.status,
+      processing: false
+    }
+  },
+
+  render () {
+    return (
+      <div class='col-lg-4 col-sm-6 mb-4'>
+        <div class={`card card-instance-details border-${this.status.level}`}>
+          <div class='card-header'>
+            <div class='d-flex'>
+              <h5 class='card-title flex-grow-1 m-0'>
+                <a href={this.instance.url} class={`text-${this.status.level}`}>
+                  {this.instance.name}
+                </a>
+              </h5>
+              <div class='dropdown'>
+                <a
+                  class='dropdown-toggle'
+                  type='button'
+                  id={`openode-instance-${this.instance.id}`}
+                  data-toggle='dropdown'
+                  aria-haspopup='true'
+                  aria-expanded='false'
+                >
+                  <i class='fa fa-cog' />
+                </a>
+                <div
+                  class='dropdown-menu dropdown-menu-right'
```

Comment:
![image](https://user-images.githubusercontent.com/8007722/73585744-23743400-4472-11ea-8910-24b40606f5cd.png)

Reply:
Gotcha, looks like the .btn class is messed up there. Will fix.